models/index.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || "development";
+const config = require(__dirname + "/../config/config.js")[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(
+    config.database,
```

Comment:
sequelize also supports taking a connection string url like postgres://username:password@hosturl/database

Reply:
i think instead of having 5 environment variables i'd prefer to consolidate

Reply:
Done

Reply:
const sequelize = new Sequelize(process.env.SQL_CONNECTION_URL, config);